crates/aptos-rosetta/src/types/objects.rs

```diff
@@ -1733,3 +1829,10 @@ pub struct ResetLockup {
     pub owner: AccountAddress,
     pub operator: AccountAddress,
 }
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct Distribute {
```

Comment:
I would call this `DistributeStakingRewards` to be a little more specific, since it could distribute other things